docs/spec/channel/version_070.md

```diff
@@ -0,0 +1,138 @@
+# Channel Spec (Version 0.7.0)
+
+## Background
+
+Starting with Version 0.7.0 all the different Channel CRDs (e.g. `InMemoryChannel` or `KafkaChannel`) are living in the `messaging.knative.dev/v1alpha1` API Group.
+
+A channel logically receives events on its input domain and forwards them to its subscribers. Below is a specification for the generic parts of each _Channel_.
+
+A typical channel consists of a _Controller_ and a _Dispatcher_ pod.
+
+## Channel Spec Parts
+
+### Control Plane
+
+Each Channel implementation is backed by its own CRD (e.g. `InMemoryChannel` or `KafkaChannel`). Unlike in earlier versions there is no concept for a _default channel_, using the different CRDs. Below is an example for a `KafkaChannel` object:
+
+```
+apiVersion: messaging.knative.dev/v1alpha1
+kind: KafkaChannel
+metadata:
+  name: kafka-channel
+spec:
+  numPartitions: 3
+  replicationFactor: 1
+```
+
+A different example for the `InMemoryChannel`:
+
+```
+apiVersion: messaging.knative.dev/v1alpha1
+  kind: InMemoryChannel
+  metadata:
+    name: my-channel
+```
+
+Each _Channel Controller_ ensures the required tasks on the backing technology are applied. In this case a Kafka topic with the desired configuration is being created, backing all messages from the channel.
```

Comment:
Do you want to define the behaviour of the `status` field to indicate that the channel has created the backing resources and is ready to receive messages?